<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>611</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ajouter un entretient</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>551</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>AGE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>PRENOM</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_AGE">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_NOM">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_PRENOM">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>50</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>120</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_SEXE">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>120</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_DATE_NAISSANCE">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>150</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_ADRESS">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>180</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_EXPERINCE">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>220</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_CIN">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>260</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>SEXE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>DATE_NAISSANCE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ADRESS</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>EXPERIENCE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>afficher les entretient</string>
     </attribute>
     <widget class="QTableView" name="tab_entretient">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>11</y>
        <width>611</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>supprimer un entretient</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_cin_sup">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_sup">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
